src/views/Print.vue

```diff
@@ -117,10 +119,10 @@ export default {
     },
     print() {
       if (this._notes === '') {
-        this._notes = 'My awesome keymap';
+        this._notes = this.$t('message.layer.empty');
```

Comment:
Should be `message.notes.empty` here.

```suggestion
        this._notes = this.$t('message.notes.empty');
```